text/0032-nft-standard.md

```diff
@@ -69,14 +71,49 @@ The metadata should have the following JSON structure:
         "type": "string",
         "description": "A URL pointing to an immutable resource with a digital asset to which this token represents"
     },
+    "collection": {
+        "type": "Object<CollectionObject>",
+        "description": "Additional details are recommended for NFT collections with multiple unique assets under one single family and symbol."
+    },
     "attributes": {
         "type": "Array<AttributeObject>",
         "description": "These are the extra attributes for the digital asset. It's an array of AttributeObject to make it as flexible as possible."
     },
+    "royalty": {
+        "type": "Object<RoyaltyObject>",
+        "description": "Object with fields that would be utilised to pay royalty to original creators in secondary sales conducted by marketplaces that adheres Hathor Metadata standards, after minting (primary sale)."
+    },
+    "external_url": {
+        "type": "string",
+        "description": "This is the URL that will appear with the asset in the marketplaces and will direct the users to view the item on your site."
+    },
+    "animation_url": {
+        "type": "string",
+        "description": "URL to a file that will be used as a preview of the NFT in the marketplaces or wallet. This is optional, and if doesn't exist, the file field will be used."
```

Comment:
"If not set, the file field may be used."

I think it's better than saying it's optional, as all other fields are also optional.

Reply:
Done